pydecorate/tests/test_decorator_agg.py

```diff
@@ -66,6 +70,13 @@ def test_colorbar(tmp_path, orientation_func_name, align_func_name, clims):
     )
 
 
+def test_add_logo():
+    img = Image.fromarray(np.zeros((200, 200, 3), dtype=np.uint8))
+    dc = DecoratorAGG(img)
+    dc.add_logo(REPO / "logos/pytroll_light_big.png")
```

Comment:
Technically shouldn't this be:

```suggestion
    dc.add_logo(REPO / "logos" / "pytroll_light_big.png")
```

to handle non-"/" systems properly?

Reply:
I think pathlib will handle it correctly for the platform either way.

Reply:
Yeah you might we right. I committed it anyway. Thanks again.